tedana/resources/external_regressor_configs/Mot12_CSF.json

```diff
@@ -0,0 +1,27 @@
+{
+    "regress_ID": "Motion12_CSF",
+    "info": "Contains 6 columns for motion, 6 for first derivatives of motion, and 1 for a CSF ROI time series.",
+    "detrend": true,
+    "calc_stats": "F",
+    "f_stats_partial_models": [
+        "Motion",
+        "CSF"
+    ],
+    "Motion": [
+        "Mot_X",
+        "Mot_Y",
+        "Mot_Z",
+        "Mot_Pitch",
+        "Mot_Roll",
+        "Mot_Yaw",
+        "Mot_d1_X",
+        "Mot_d1_Y",
+        "Mot_d1_Z",
+        "Mot_d1_Pitch",
+        "Mot_d1_Roll",
+        "Mot_d1_Yaw"
```

Comment:
Are these the column names produced by AFNI?

Reply:
No. I think they were a tweak of the names you used in an earlier version. Fine changing, if you want to align naming with at least one program.

Reply:
Awesome. I'll look up the fMRIPrep naming and propose a change soon then.

Reply:
Actually, are the configs still used in the current version of the PR?

Reply:
No! This info is now only used from the `decision_tree/demo...` files. That makes this an even easier fix.